diff --git a/node_modules/@umijs/bundler-esbuild/dist/build.js b/node_modules/@umijs/bundler-esbuild/dist/build.js
index b95b337..bbb347e 100644
--- a/node_modules/@umijs/bundler-esbuild/dist/build.js
+++ b/node_modules/@umijs/bundler-esbuild/dist/build.js
@@ -19,6 +19,7 @@ async function build(opts) {
     return await (0, esbuild_1.build)({
         entryPoints: opts.entry,
         bundle: true,
+        banner: { js: '"use strict";' },
         format: opts.format || 'iife',
         logLevel: 'error',
         // splitting: true,
diff --git a/node_modules/@umijs/bundler-esbuild/dist/plugins/style.js b/node_modules/@umijs/bundler-esbuild/dist/plugins/style.js
index 444c9c9..6d5748a 100644
--- a/node_modules/@umijs/bundler-esbuild/dist/plugins/style.js
+++ b/node_modules/@umijs/bundler-esbuild/dist/plugins/style.js
@@ -26,6 +26,7 @@ function style({ minify = true, charset = 'utf8', inlineStyle, config, } = {}) {
                     '.svg': 'dataurl',
                     // file ?
                     '.ttf': 'dataurl',
+                    '.eot': 'dataurl',
                 },
             };
             onResolve({ filter: /\.css$/, namespace: 'file' }, (args) => {
